README.md

```diff
@@ -16,6 +16,40 @@ n/a
   - bcrypt
   - jsonwebtoken
 
+### How to initialize/run your application (where applicable)
+* npm start
+### How to use your library (where applicable)
+n/a
+### Tests
+* npm install jest
+* npm install @code-fellows/supergoose
+* npm test  
+### UML
+![UML](uml-diagram.png)
+
+---
+
+# LAB - Class 12   
+## Project: OAuth
+### Author: Jennifer Chinzi
+### Links and Resources
+n/a  
+### Setup
+* create .env
+  - PORT=####
+  - MONGODB_URI=mongodb://localhost:27017/DB_NAME
+  - SECRET=somesecretstring
+  - GITHUB_CLIENT_ID=yourGitHubClientID
+  - GITHUB_CLIENT_SECRET=yourGitHubClientSecret
```

Comment:
This is really great Jen, thanks for putting everything that someone else will need in their .env